cmd/prepare-storage.go

```diff
@@ -238,8 +234,26 @@ func waitForFormatXL(ctx context.Context, firstDisk bool, endpoints EndpointList
 	retryTimerCh := newRetryTimerSimple(doneCh)
 	for {
 		select {
-		case _ = <-retryTimerCh:
-			format, err := connectLoadInitFormats(firstDisk, endpoints, setCount, disksPerSet)
+		case retry := <-retryTimerCh:
+			// Initialize all storage disks
+			storageDisks, err := initStorageDisks(endpoints)
+			if err != nil {
+				return nil, err
+			}
+			defer closeStorageDisks(storageDisks)
+
+			// Connect to all storage disks, a connection failure will be
+			// only logged after some retries.
+			for _, disk := range storageDisks {
+				if disk != nil {
+					connectErr := disk.Connect()
+					if connectErr != nil && retry >= 5 {
+						logger.Info("Unable to connect to %s: %v\n", disk.String(), connectErr.Error())
```

Comment:
No need to do disk.String() since there is a stringer interface, fmt will automatically print String() for `%s`